include/c_utilities/get_env.h

```diff
@@ -0,0 +1,35 @@
+// Copyright 2017 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef C_UTILITIES__GET_ENV_H_
+#define C_UTILITIES__GET_ENV_H_
+
+#if __cplusplus
+extern "C"
+{
+#endif
+
+#include <stdbool.h>
+
+#include "c_utilities/visibility_control.h"
+
+C_UTILITIES_PUBLIC
+bool
+get_env(const char * env_name, const char ** env_value);
```

Comment:
Also there are no tests for this. They could be scavenged from:

https://github.com/ros2/rmw/pull/87/files#diff-4389729cf437a8deb24b70ef12a2b57e